app/Services/Caches/CampaignCacheService.php

```diff
@@ -245,6 +245,9 @@ public function themes(): string|bool
         /** @var CampaignPlugin $plugin */
         foreach ($plugins as $plugin) {
             $theme .= "/** plugin: " . e($plugin->name) . " #" . e($plugin->version->version) . " **/\n";
```

Comment:
Would be great to only have a comment in the css if the fonts field isn't empty. At the moment it's adding 16 useless lines on my local config :P